ArcGISRuntimeSDKQt_CppSamples/Layers/DisplayOgcApiCollection/main.cpp

```diff
@@ -0,0 +1,67 @@
+// Copyright 2021 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifdef Q_OS_WIN
+#include <Windows.h>
+#endif
+
+#include "DisplayOgcApiCollection.h"
+
+#include "ArcGISRuntimeEnvironment.h"
+
+#include <QDir>
+#include <QGuiApplication>
+#include <QQmlApplicationEngine>
+
+int main(int argc, char *argv[])
+{
+  QGuiApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
+  QGuiApplication app(argc, argv);
+  app.setApplicationName(QStringLiteral("DisplayOgcApiCollection - C++"));
+
+  // Use of Esri location services, including basemaps and geocoding,
+  // requires authentication using either an ArcGIS identity or an API Key.
+  // 1. ArcGIS identity: An ArcGIS named user account that is a member of an
+  //    organization in ArcGIS Online or ArcGIS Enterprise.
+  // 2. API key: A permanent key that gives your application access to Esri
+  //    location services. Visit your ArcGIS Developers Dashboard create a new
+  //    API keys or access an existing API key.
+  const QString apiKey = QStringLiteral("");
```

Comment:
```suggestion
  const QString apiKey = QString("");
```

This came up recently. It should be QString since an empty QStringLiteral will produce a clazy warning.
https://github.com/KDE/clazy/blob/master/docs/checks/README-empty-qstringliteral.md